src/global_helpers.R

```diff
@@ -575,7 +575,8 @@ get_product_info <- function(network, domain=NULL, status_level, get_statuses){
 #. handle_errors
 prodcode_from_prodname_ms <- function(prodname_ms){
 
-    prodcode = strsplit(prodname_ms, '_')[[1]][2]
+    name_length = length(strsplit(prodname_ms, '_')[[1]])
+    prodcode = strsplit(prodname_ms, '_')[[1]][max(name_length)]
```

Comment:
nice catch. prodcodes that contain underscores would have been subdivided if we always grabbed the element at the second index. however, always grabbing the last won't do either. we want everything after the first. Say we encounter a chemistry prodcode "A_001". The prodname_ms would be "chemistry_A_001", and the stringsplit would separate the A and the 001, which should stay together. Also, this function isn't written to accept vector arguments, so name_length will always be a single integer. Using `max` is unnecessary.